docs/src/main/sphinx/release/release-423.md

```diff
@@ -0,0 +1,134 @@
+# Release 423 (9 Aug 2023)
+
+## General
+
+* Add support for renaming nested fields in a column with a `RENAME COLUMN`
+  statement. ({issue}`16757`)
+  * Add support for setting the type of a nested field in a column with a `SET
+  DATA TYPE` statement. ({issue}`16959`)
+* Add support for column comments on materialized views. ({issue}`18016`)
+* Improve performance of `INSERT` and `CREATE TABLE AS ... SELECT` queries. ({issue}`18212`)
+* Improve performance when planning queries involving multiple window functions. ({issue}`18491`)
+* Improve optimizer estimates for queries with a `BETWEEN` clause. ({issue}`18501`)
+* Improve performance of queries containing redundant `ORDER BY` clauses in
+  views or `WITH` clauses. This may affect the semantics of queries that
+  incorrectly rely on implementation-specific behavior. The old behavior can be
+  restored via the `skip_redundant_sort` session property or the
+  `optimizer.skip-redundant-sort` configuration property. ({issue}`18159`)
+* Reduce default values for the `task.partitioned-writer-count` and
+  `task.scale-writers.max-writer-count` configuration properties to reduce the
+  memory requirements of queries that write data. ({issue}`18488`)
+* Reduce DNS lookup overhead on NodeScheduler. ({issue}`18365`)
+* Disallow coercing `timestamp` types to `varchar` for historical dates. ({issue}`18004`)
```

Comment:
* What's a "historical date"? 
* There are no implicit casts between timestamp and varchar in the engine -- this change is specific to Hive, but in what context does it apply? E.g., for partition columns? When column metadata doesn't match data in files, etc?

Reply:
Have to look at the PR some more to see if I can get that info.

Reply:
Historical dates are before 1900 in this PR. Not sure about what context .. seems to be generic, at least from just looking at the PR.

Reply:
Then we should call it out as "dates before xxx".

Reply:
Done